src/applications/mhv-landing-page/tests/hooks/useAccountCreationApi.unit.spec.js

```diff
@@ -0,0 +1,64 @@
+import { expect } from 'chai';
+import sinon from 'sinon';
+import React from 'react';
+import { renderWithStoreAndRouter } from '@department-of-veterans-affairs/platform-testing/react-testing-library-helpers';
+import { useAccountCreationApi } from '../../hooks';
+
```

Comment:
```suggestion
import {
  fetchAccountStatus,
  fetchAccountStatusSuccess,
} from '../reducers/account';
```

This would be needed for the other suggestions